src/Features/Core/Portable/CodeRefactorings/MoveType/AbstractMoveTypeService.RenameFileEditor.cs

```diff
@@ -24,14 +24,12 @@ internal override Task<ImmutableArray<CodeActionOperation>> GetOperationsAsync()
             /// </summary>
             private ImmutableArray<CodeActionOperation> RenameFileToMatchTypeName()
             {
-                var oldDocument = SemanticDocument.Document;
-                var newDocumentId = DocumentId.CreateNewId(oldDocument.Project.Id, FileName);
+                var documentId = SemanticDocument.Document.Id;
+                var oldSolution = SemanticDocument.Document.Project.Solution;
+                var newSolution = oldSolution.WithDocumentName(documentId, FileName);
 
                 return ImmutableArray.Create<CodeActionOperation>(
-                    new RenameDocumentOperation(
-                        oldDocument.Id, newDocumentId,
-                        FileName, SemanticDocument.Text),
-                    new OpenDocumentOperation(newDocumentId, activateIfAlreadyOpen: true));
+                    new ApplyChangesOperation(newSolution));
```

Comment:
is this a change in behavior?  if so, why?

Reply:
It should not be, but I'll manually check for regressions again (you've made me nervous).

Reply:
it looks like we'd miss the OpenDocumentOperation.

Reply:
That was needed because the action of renaming removed/added the file, which could close the editor. Now, the file actually stays open (and you keep your breakpoints, etc.). I don't think it is needed anymore.

Reply:
Great.  Thanks for the clarification.  It's possibly worth having this be stated in a comment here.  But it's not hugely necessary.

Reply:
Double checked end-to-end -- no regressions.

Reply:
Thanks!

Reply:
Back to your original question, I should have said YES, it's a change in behavior -- it actually works the way you'd want it to now. :)